### DISCUSSION TITLE
Constants sometimes don't create

### DISCUSSION THREAD

**Discussion by oehm-smith** [postmanlabs/postman-flows#27] [Defect]
![image](https://user-images.githubusercontent.com/2556595/137065032-6e4b37f2-f14e-4b70-b697-0272e3de5e6d.png)

I have been using this pattern to report the success or failure of a test.  

This is my 2nd test and the 2nd time i've created these constants.  In the first test it worked fine.  But for my 2nd test, the SUCCESS constant was not created in the "create variable" block.  My terminal only contained `{}` and the `block > information > Output generated` only had `$ Object`.  I added 2 more constants in the same "create variable" block to get it to work.  Now that `Output generated` contains:

![image](https://user-images.githubusercontent.com/2556595/137065523-ba23e626-c4ff-4fbf-8331-374e4e05824c.png)

**Comment by samay-v** [comment]
Hey @oehm-smith, Are you using the Postman Desktop app or the webapp?
We released a fix for a similar issue in [Release 21.10.3](https://github.com/postmanlabs/postman-flows/discussions/10), which would not have been reflected in the desktop app yet. Let us know if this is reproducible for you in the [webapp](go.postman.co) as well?

**Reply by oehm-smith** [reply]
This problem was in the Desktop app.  Cool I look forward to the change and will let you know if I see it in the webapp in the meantime (or Desktop after that fix is deployed).
